doc/rst/source/grdcut.rst

```diff
@@ -57,6 +58,16 @@ Required Arguments
 Optional Arguments
 ------------------
 
+.. _-D:
+
+**-D**
+    A "dry run": Simply report the region and increment of what would be the
+    extracted grid given the selected options.  No grid is created (**-G** is disallowed)
+    and instead we write a single data record with *west east south north xinc yinc*
```

Comment:
Should add what the default units are for *xinc* and *yinc* since this is not reported.

Reply:
Do we need to say? They are always in the same units as w/e/s/n unless there is an explicit unit like m or s.  That is only for -D+t; without +t they are all numbers so no m or s possible.

Reply:
OK, that's fine.